chameleonultragui/lib/helpers/general.dart

```diff
@@ -131,15 +134,15 @@ Color hexToColor(String hex) {
 
 String platformToPath() {
   if (Platform.isAndroid) {
-    return "android";
+    return "Android";
```

Comment:
This is used for tests workaround, not displayed to user. Revert changes

Reply:
Done